src/components/map/Toolbar.tsx

```diff
@@ -0,0 +1,75 @@
+import * as s from './toolbar.scss';
+import React, { Component } from 'react';
+import { FiEdit, FiCopy, FiPlusSquare, FiPrinter } from 'react-icons/fi';
+import ToolbarIcon from './ToolbarIcon';
+
+interface ToolbarState {
+    editModeIsActive: boolean;
```

Comment:
isEditModeActive? (We have isMapFullScreen)

I think better might be "isRouteEditable" but not sure yet because we aren't that far yet. So no need to change.